Deploy-CAM.ps1

```diff
@@ -331,8 +339,9 @@ function Register-RemoteWorkstation()
 		subscriptionId = $subscription
 	}
 
-	$machineRegistrationError = ""
 	for($idx = 0; $idx -lt $retryCount; $idx++) {
+		$machineRegistrationError = ""
```

Comment:
Are we sure that it's not used outside of the for loop as well? (I didn't check.)

Reply:
the same as above